src/scheduler/workers/unpublish/day/changeState/unpublishListItems.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from "server/models/db/prisma-client";
+import { EVENTS } from "server/models/listItem/listItemEvent";
+
+export async function unpublishListItems(ids: number[], reference?: string) {
+  return ids.map(async (listItemId: number) => {
+    return await prisma.listItem.update({
+      where: {
+        id: listItemId,
+      },
+      data: {
+        status: "ANNUAL_REVIEW_OVERDUE",
+        isAnnualReview: false,
+        isPublished: false,
+        history: {
+          create: EVENTS.ANNUAL_REVIEW_OVERDUE(reference),
```

Comment:
I originally had that in mind but then noticed the status is also ANNUAL_REVIEW_OVERDUE rather than UNPUBLISHED and the view model uses this to display the correct label.  I was going for consistency to keep inline with the status change.  The main difference between ANNUAL_REVIEW_OVERDUE vs UNPUBLISHED and CHECK_ANNUAL_REVIEW vs EDITED is the that isAnnualReview is always when the status is in ANNUAL_REVIEW_OVERDUE / UNPUBLISHED so there's nothing to differentiate between them.  The events / history could be queried but I think using the status is probably a cleaner way of querying the data.  Have a think and we can debate this one.

Reply:
I don't think it needs its own event type but is fine with. ignoring that, I still need the additional unpublish event. An event describing the "act" of unpublishing still needs to be added. ANNUAL_REVIEW_OVERDUE event only describes that someone has not responded in X weeks and that the ListIten.status has changed to ANNUAL_REVIEW_OVERDUE. 

I want an easy way to find out why something was unpublished (isPublished turned off). if it's only described by ANNUAL_REVIEW_OVERDUE, I need to add another condition and search over 3 events (publish, unpublish, annual_review_overdue).

There is currently no other mechanism that the system has to unpublish something. The only other way something is unpublished is by a user, which is why I mention do not pass in user id. There is also no reason why we can't add `isAnnualReview` to either the jsonData or as a column but we can resolve that later. 

tl;dr, you can keep the additional event type, but please add an unpublish event. 

```
history: {
  create: [EVENTS.UNPUBLISH(..), EVENTS.ANNUAL_REVIEW_OVERDUE(..)]
}
```

addendum
it's slightly different for CHECK_ANNUAL_REVIEW / EDITED. they are functionally the same. again, can be resolved by `isAnnualReview` in json or column.